documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Try not to recommend libraries that are "competition". If there's a good reason to recommend SKL or one of the others, do so with context (e.g. "SKL supports $x algo, go check it out if you are interested.")</p><p> </p><p>I think the second para can go too. We don't talk about how old things are (<a href="https://developers.google.com/style/timeless-documentation" target="_blank">documentation should be timeless</a>), and we've already introduced TFDF.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>